### PR TITLE
Ed/post state of urbit

### PR DESCRIPTION
**Pull request by urcades**

### CODE REVIEW

**Comment by matildepark** [comment]
idk what 'assigned' means here

**Comment by urcades** [comment]
@matildepark sorry, wrong field my bad

File: content/blog/20200929-state-of-urbit.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
++++
+title = "Late 2020 Progress Update: OS 1 -> OS 1.N"
+date = 2020-09-29
+description = "When we announced OS 1, in April, we started to disappear into Urbit. Since then, we’ve been living on Urbit like we never have before."
+[extra]
+author = "Galen Wolfe-Pauly"
+ship = "~ravmel-ropdyl"
+image ="https://storage.googleapis.com/media.urbit.org/site/posts/essays/state-of-urbit-1.JPG"
```
Comment (matildepark) [inline content/blog/20200929-state-of-urbit.md]: Does this build without the space?

Comment (urcades) [reply content/blog/20200929-state-of-urbit.md]: It seems to!

**Review by matildepark — APPROVED** [review]

**Pull request merged**